src/controllers/post.js

```diff
@@ -10,11 +11,40 @@ export const create = async (req, res) => {
     const postToCreate = await Post.fromJson(req.body)
     postToCreate.userId = req.user.id
     const post = await postToCreate.save()
+
     return sendDataResponse(res, 201, post)
   } catch (err) {
     return sendDataResponse(res, 400, { err: err.message })
   }
 }
+
+export const createComment = async (req, res) => {
+  const postId = +req.query.postId
+  const { content } = req.body
+  if (!content) {
+    return sendDataResponse(res, 400, { content: 'Must provide content' })
+  }
+  try {
+    const commentToCreate = await Comment.fromJson(req.body)
+    commentToCreate.userId = req.user.id
+    commentToCreate.profileId = req.user.id
+    commentToCreate.postId = postId
+    const comment = await commentToCreate.save()
+    return sendDataResponse(res, 201, comment)
+  } catch (err) {
+    return sendDataResponse(res, 400, { err: err.message })
+  }
+}
+export const findAllComments = async (req, res) => {
+  try {
```

Comment:
What's the reasoning for the try/catch block here? What might throw an error?

Reply:
Originally was not working but we modified the try block to throw a new error if no comments are found.

Reply:
![Catch error insomnia](https://user-images.githubusercontent.com/98411345/176479294-220f6995-33b5-4aa3-8047-b01db5546fd5.png)